client/hospital.lua

```diff
@@ -157,6 +161,24 @@ if config.useTarget then
                             icon = "fas fa-clipboard",
                             label = Lang:t('text.bed'),
                             distance = 1.5,
+                        },
+                        {
+                            canInteract = function()
+                                if QBX.PlayerData.job.name == 'ambulance' then
+                                    return true
+                                end
+                                return false
+                            end,
+                            onSelect = function()
+                                local player = GetClosestPlayer()
+                                if player then
+                                    local playerId = GetPlayerServerId(player)
+                                    TriggerServerEvent('hospital:server:putPlayerInBed', playerId, hospitalName, i)
```

Comment:
It doesn't look like this server event does anything but call this client event to put a player in bed. Why not just call the function putPlayerInBed here directly?